src/version.js

```diff
@@ -0,0 +1 @@
+export const APPLICATION_VERSION = `WEB v${require("../package.json").version}`;
```

Comment:
I would do this through the vue config (e.g. something like this: https://medium.com/hceverything/how-to-show-your-app-version-from-package-json-in-your-vue-application-11e882b97d8c). Reading the package.json in actual application code doesn't feel that nice.

Also, I would just write `Accentor Web v${version}`.